etcdmain/help.go

```diff
@@ -183,5 +183,7 @@ auth flags:
 experimental flags:
 	--experimental-corrupt-check-time '0s'
 	        duration of time between cluster corruption check passes.
+	--experimental-enable-v2v3 ''
+		store v2 keys under the given v3 prefix instead of v2 backend.
```

Comment:
serve v2 requests through v3 backend?

we not only store v2 keys under v3, but also get keys from v3. this sounds that users can still get v2 keys from the previous v2 store.